sdk/maps/maps-geolocation-rest/package.json

```diff
@@ -0,0 +1,129 @@
+    "integration-test:node": "dev-tool run test:node-js-input -- --timeout 5000000 'dist-esm/test/**/*.spec.js'",
+    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
+    "lint:fix": "eslint package.json api-extractor.json src test --ext .ts --fix --fix-type [problem,suggestion]",
+    "lint": "eslint package.json api-extractor.json src test --ext .ts",
+    "pack": "npm pack 2>&1",
+    "test:browser": "npm run clean && npm run build:test && npm run unit-test:browser",
+    "test:node": "npm run clean && npm run build:test && npm run unit-test:node",
+    "test": "npm run clean && npm run build:test && npm run unit-test",
+    "unit-test": "npm run unit-test:node && npm run unit-test:browser",
+    "unit-test:node": "dev-tool run test:node-ts-input -- --timeout 1200000 --exclude 'test/**/browser/*.spec.ts' 'test/**/*.spec.ts'",
+    "unit-test:browser": "dev-tool run test:browser",
+    "build": "npm run clean && tsc -p . && dev-tool run bundle && mkdirp ./review && api-extractor run --local"
+  },
+  "sideEffects": false,
+  "autoPublish": false,
+  "dependencies": {
+    "@azure/core-auth": "^1.3.0",
+    "@azure-rest/core-client": "1.0.0-beta.10",
+    "@azure/core-rest-pipeline": "^1.8.0",
+    "@azure/maps-common": "^1.0.0-beta.2",
+    "tslib": "^2.2.0"
+  },
+  "devDependencies": {
+    "@microsoft/api-extractor": "^7.31.1",
+    "autorest": "latest",
+    "@types/node": "^14.0.0",
+    "dotenv": "^8.2.0",
+    "eslint": "^8.0.0",
+    "mkdirp": "^1.0.4",
+    "prettier": "2.2.1",
+    "rimraf": "^3.0.0",
+    "source-map-support": "^0.5.9",
+    "typescript": "~4.6.0",
+    "@azure/dev-tool": "^1.0.0",
+    "@azure/eslint-plugin-azure-sdk": "^3.0.0",
+    "@azure-tools/test-credential": "^1.0.0",
+    "@azure/identity": "^2.0.1",
+    "@azure-tools/test-recorder": "^2.0.0",
+    "mocha": "^7.1.1",
+    "@types/mocha": "^7.0.2",
+    "mocha-junit-reporter": "^1.18.0",
+    "cross-env": "^7.0.2",
+    "@types/chai": "^4.2.8",
+    "chai": "^4.2.0",
+    "karma-chrome-launcher": "^3.0.0",
+    "karma-coverage": "^2.0.0",
+    "karma-edge-launcher": "^0.4.2",
+    "karma-env-preprocessor": "^0.1.1",
+    "karma-firefox-launcher": "^1.1.0",
+    "karma-ie-launcher": "^1.0.0",
+    "karma-junit-reporter": "^2.0.1",
+    "karma-mocha-reporter": "^2.2.5",
+    "karma-mocha": "^2.0.1",
+    "karma-source-map-support": "~1.4.0",
+    "karma-sourcemap-loader": "^0.3.8",
+    "karma": "^6.2.0",
+    "nyc": "^14.0.0",
+    "@azure/test-utils": "~1.0.0"
+  },
+  "homepage": "https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/maps/maps-geolocation-rest/README.md",
+  "//metadata": {
+    "constantPaths": [
+      {
+        "path": "swagger/README.md",
+        "prefix": "package-version"
+      }
+    ]
+  },
+  "browser": {
+    "./dist-esm/test/public/utils/env.js": "./dist-esm/test/public/utils/env.browser.js"
+  },
+  "//sampleConfiguration": {
+    "productName": "MapsGeolocationClient",
```

Comment:
Do you mind adding this property inside //sampleConfiguration?  `"disableDocsMs": true`